exp/addons/controllers/clusterresourceset_controller_test.go

```diff
@@ -72,13 +70,14 @@ kind: ConfigMap
 apiVersion: v1`,
 			},
 		}
-
-		testConfigmap2 := &corev1.ConfigMap{
+		testEnv.Create(ctx, testConfigmap)
+		testSecret := &corev1.Secret{
 			ObjectMeta: metav1.ObjectMeta{
-				Name:      configmap2Name,
+				Name:      secretName,
 				Namespace: defaultNamespaceName,
 			},
-			Data: map[string]string{
+			Type: "addons.cluster.x-k8s.io/resource-set",
+			StringData: map[string]string{
 				"cm": `metadata:
 kind: ConfigMap
```

Comment:
Should this be `Secret` or does it not matter what the contents is?

Reply:
We are testing if we can also use Secrets as resources now, its internal does not matter.